<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1526222824316" stop="1526222826461">
	<name>Validating the table</name>
	<title>Validating the table</title>
	<test-cases>
		<test-case start="1526222824316" stop="1526222826461" status="failed">
			<name>validate table</name>
			<title>validate table</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="011c65ce-a29e-48bf-9dfc-3320af4e8107-attachment.png" type="image/png" size="76986"/>
			</attachments>
			<failure>
				<message>Failed: by.class is not a function</message>
				<stack-trace>TypeError: by.class is not a function
    at C:\Users\123\WebstormProjects\ProtractorTest\tabletest.js:11:45
    at ManagedPromise.invokeCallback_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: Run it(&quot;validate table&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
    at ControlFlow.emit (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
    at ControlFlow.shutdown_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
    at shutdownTask_.MicroTask (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2599:53)
From asynchronous test: 
Error
    at Suite.&lt;anonymous&gt; (C:\Users\123\WebstormProjects\ProtractorTest\tabletest.js:5:3)
    at addSpecsToSuite (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\123\WebstormProjects\ProtractorTest\tabletest.js:1:63)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>